DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/mmhost/src/ddr_hosts_model.cpp

```diff
@@ -0,0 +1,115 @@
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <sycl/sycl.hpp>
+#include <sycl/ext/oneapi/annotated_arg/annotated_ptr.hpp>
+
+#include "exception_handler.hpp"
+
+using namespace sycl;
+using namespace ext::oneapi::experimental;
+
+using usm_buffer_location =
+    ext::intel::experimental::property::usm::buffer_location;
+
+constexpr int kBL1 = 1;
+constexpr int kBL2 = 2;
+
+struct DDRIP{
+  annotated_ptr<int, decltype(properties{
+          buffer_location<kBL1>,
+          maxburst<8>,
+          dwidth<256>,
+          alignment<32>
+          })> x; 
+  annotated_ptr<int, decltype(properties{
+          buffer_location<kBL1>,
+          maxburst<8>,
+          dwidth<256>,
+          alignment<32>
+          })> y;
+  annotated_ptr<int, decltype(properties{
+          buffer_location<kBL2>,
+          maxburst<8>,
+          dwidth<256>,
+          alignment<32>
+          })> z;  
+
+  int size;
+
+  void operator()() const {
+    for(int i = 0; i < size; i++){
+      int add = x[i] + y[i];
+      z[i] = add;
+    }
+  }
+};
+
+int main(void){
+#if FPGA_SIMULATOR
+  auto selector = sycl::ext::intel::fpga_simulator_selector_v;
+#elif FPGA_HARDWARE
+  auto selector = sycl::ext::intel::fpga_selector_v;
+#else  // #if FPGA_EMULATOR
+  auto selector = sycl::ext::intel::fpga_emulator_selector_v;
+#endif
+
+  bool passed = true;
+
+  try {
+    // create the device queue
+    sycl::queue q(selector, fpga_tools::exception_handler,
+                  sycl::property::queue::enable_profiling{});
```

Comment:
Do we still need to do profiling? If not, remove this from all files.

Reply:
@whitepau Do we also still need to do profiling? I am not sure about this one too